spec/support/page_objects/sections/home_hero.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require_relative "base"
+
+module PageObjects
+  module Sections
+    class HomeHero < PageObjects::Sections::Base
```

Comment:
What about using:

```ruby
module PageObjects
  class HomeHeroSection < PageObjects::Sections::Base
    ...
  end
end
```

Reply:
I think it would be good to keep the separation between pages and sections so it's easier to find which is which.